packages/cozy-harvest-lib/src/Manifest.js

```diff
@@ -38,6 +39,14 @@ const defaultFieldsValues = fields => {
   )
 }
 
+/**
+ * Returns the key for the field having the role=identifier attribute
+ * @param  {Object} fields Konnector fields
+ * @return {[type]}        The key for the identifier field, example 'login'
+ */
+const getIdentifier = (fields = {}) =>
+  findKey(sanitizeIdentifier(fields), field => field.role === IDENTIFIER)
```

Comment:
Is it the role of the getter to sanitize ? I think inputs should be sanitized/normalized as early as possible so that this method does not have to sanitize by itself.

Reply:
I agree with sanitizing early, but here I am not sure. Actually getIdentifier could not return any identifier if fields parameter is not sanitized.

For now, fields are sanitized in AccountForm at rendering.

Maybe it indicates a bad conception here, our konnector should be sanitized in the TriggerManager. It would imply that our Manifest helper should better be a konnector helper/model.

But event there, I could not be sure that calling:

```js
KonnectorModel.fields.getIdentifier(konnector.fields)
```

will return me identifier if `konnectors.fields` are not sanitized.